sdk/core/azure-core/README.md

```diff
@@ -1,7 +1,13 @@
 
 # Azure Core Library
 
-Azure core library defines basic exceptions and shared modules those are needed when you use client libraries. If you are a client library developer, please reference [client library developer reference](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md) for more information.
+Azure core library defines basic exceptions and shared modules those are needed when you use client libraries. 
+
+As an end user, you don't need to manually install azure-core because it will be installed automatically when you install other SDKs. 
+
+If you are a client library developer, please reference [client library developer reference](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md) for more information.
+
+[Source code]() | [Package (Pypi)][package] | [API reference documentation]()
```

Comment:
Do we need a couple more links here?

Love the text!

Reply:
I checked other SDKs. Some of them also have Product Doc link and Samples link. But neither of them is applicable for the core.